plugins/base/src/main/kotlin/parsers/moduleAndPackage/ModuleAndPackageDocumentationSource.kt

```diff
@@ -0,0 +1,17 @@
+package org.jetbrains.dokka.base.parsers.moduleAndPackage
+
+import java.io.File
+
+internal abstract class ModuleAndPackageDocumentationSource {
+    abstract val sourceDescription: String
+    abstract val documentation: String
+
+    override fun toString(): String {
+        return sourceDescription
```

Comment:
Maybe a `=` here

Reply:
Best example where it actually makes sense! ☺️ 
Changed it ✅